docs/source/api_references.rst

```diff
@@ -137,8 +137,14 @@ Profiler API
 .. autosummary::
     :toctree: api
     :nosignatures:
+    :template: classtemplate.rst
+
+    AdvancedProfiler
+    BaseProfiler
+    PassThroughProfiler
+    PyTorchProfiler
+    SimpleProfiler
```

Comment:
should the abstract profiler also be documented here?

Reply:
Yes. Done !